tests/unit/utils/slugify-helpers.unit.spec.ts

```
@@ -0,0 +1,181 @@
+    })
+
+    it('devrait gérer les caractères avec cédille et tréma', () => {
+      expect(slugify('Français & Naïve', FRENCH_SLUGIFY_CONFIG))
+        .toBe('francais-et-naive')
+    })
+  })
+
+  describe('Limites et cas extrêmes', () => {
+    it('devrait gérer les très longues chaînes', () => {
+      const longString = 'A'.repeat(200) + ' & ' + 'B'.repeat(200)
+      const result = slugify(longString, FRENCH_SLUGIFY_CONFIG)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)